.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -11,12 +11,19 @@ assignees: ''
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
 Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
 
-This repository is **ONLY** for PowerShell Core 6 and PowerShell 7+ issues.
+This repository is **ONLY** for PowerShell 7+ issues:
 
-- Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
-- Search the existing issues.
-- Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
-- Refer to the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6).
+  - If you're seeing behavior that deviates from the intended, documented behavior, report it here, assuming that you can reproduce it in the [latest released version](https://github.com/PowerShell/PowerShell/releases) and that it hasn't already been reported (please search the existing issues).
+  
```

Comment:
I think we should recommend to download _daily_ build with latest fixes.

Reply:
Agreed re long instructions.

The main reason for creating this PR was to direct people to more appropriate forums, if their problem isn't an actual bug or at least _potential_ bug based on a solid understanding of the feature involved and its intended behavior.

Conversely, we don't want to make it too hard for people who do have an actual problem with PowerShell itself to report, and I think requiring use of the latest daily build is too much to ask.

I think we should allow at least the latest stable and preview versions, and possibly even all versions that are officially supported at the time of reporting.
Ideally, the actual version numbers should be explicitly enumerated, to save users a trip to another page, though for that we'd have to an automated process of updating these numbers.

Reply:
>  I think requiring use of the latest daily build is too much to ask.

But we ask often this in issues :-)

Reply:
We can _recommend_  it, but we should make clear that we don't _require_ it.

If a bug report is well written and contains simple repro steps, it's easy for others to test the behavior on newer versions.

Conversely, user may decide not to bother reporting, if they think installing the latest daily build is a requirement.

Reply:
> Conversely, user may decide not to bother reporting, if they think installing the latest daily build is a requirement.

Why not ask to make the right move? Or even demand it? This is an open source project and no one is obliged to do this check instead. If we get one less bad report then we will have more time for good reports that these reports deserve.

Reply:
My recommendation: Ask: yes. Demand: no. 

The far bigger problems are:

*  The _quality_ of the reports - people reporting/asking about things as if this were a general PowerShell support forum.

   * We want to direct them to the appropriate forums instead.

* Posting reports without having searched for an existing report or not having found one due to the specific search terms used.

  * We want to encourage prior searching, though I suspect compliance will be low.

  * There was an experimental GitHub feature for a while that showed you possible duplicates as you typed your report's title, but it looks like they ended up not implementing it, which is a shame, because it is the most convenient way to discover duplicates, assuming the dynamic search works well.

By contrast, a high-quality report, even if it isn't based on the very latest version, is still useful - assuming that they've searched existing issues first to prevent duplicates. If there is no duplicate, chances are the bug is still present in the latest version.